apps/test/unit/templates/studioHomepages/JoinSectionNotificationsTest.jsx

```diff
@@ -79,6 +86,12 @@ describe('JoinSectionNotifications', () => {
     expect(wrapper.find('JoinSectionOwnedNotification')).to.have.lengthOf(1);
   });
 
+  it('renders correct component when section is at capacity (already has 500 students)', () => {
+    let wrapper = shallow(<JoinSectionNotifications {...AT_CAPACITY} />);
+    console.log(wrapper.debug());
+    expect(wrapper.find('JoinSectionFullNotification')).to.have.lengthOf(1);
```

Comment:
Can we add an extra assertion in other tests to make sure this element isn't showing when it shouldn't?

Reply:
I went ahead and added that assertion to each of the other tests.

Is it better to write a testing helper that might test all of the variations, and we can distinguish which should be true of the lot? Otherwise, we are testing the full notification isn't rendering when it isn't supposed to, but we don't make that check for the other variations...

Happy to make that update if that feels appropriate!

Reply:
Some kind of helper function does sound ideal, but sounds like a considerable scope increase (unless you think otherwise). I'm fine leaving this PR as-is. Thanks for addressing!